docs/adr/index.md

```diff
@@ -0,0 +1,5 @@
+# Architectural Decision Log

```

Comment:
This file will need to be overridden in the community repo.

Reply:
It's only specific files that GitHub defines that are picked up for all repos, and this isn't one of them.

Were you expecting all the files from here to show in all repos?

Reply:
I guess I don't understand how this file will be used, then.

Reply:
We may, in future, copy the template to specific repos.
I'd like all the discussion around the contense of this repo to be had on the community repo.
This file aims to indicate that this repo shouldn't have ADRs.